### DISCUSSION TITLE
Oracle DB to Postgres data migration

### DISCUSSION THREAD

**Discussion by vbhaskaran003** [apache/airflow#35371] [General]
Hi all, we are planning to migrate data from onprem oracle DB to AWS Postgres . Can anyone help me if any inbuilt operator available for the same.

**Comment by Taragolis** [comment]
IMO, [AWS Database Migration Service Operators](https://airflow.apache.org/docs/apache-airflow-providers-amazon/stable/operators/dms.html) from Amazon Provider, as well as [AWS DMS](https://docs.aws.amazon.com/dms/) service looks like suits to your requirements
